### ISSUE TITLE
Limit number of sites / hopping cycles based upon user level 

### ISSUE THREAD

**Issue by jaygoodswp** [jaygoodswp/PHS#65]

**Comment by TMM21042** [comment]
@jaygoodswp at the back end, can the code be written to have administrative flexibility to configure subscription levels based on a combination of three variables.  The tier configuration variables are: 1) number of login sites, 2) the number of rules, and 3) the number of hopping cycles.  For example, we would be able to easily adjust a tier of the PHS to be one of several combinations.  Combination 1 might be 5 online sites, one rule (news source and word), and daily hopping.  Combination 2 might be 5 online sites XX rules, and XX hopping cycles, etc.  An administrative page for configuring the PHS tiers of the subscription is needed since we don't yet know which combination(s) will be used and we may need repeated adjustments.

Also, we may limit the rules and hopping cycles to certain choices at the various subscription tiers.  For example a we may limit a low tier to one rule (specifically, and administrative function to allow one news source, one title and one word), but a higher tier may allow a more sophisticated rule (like the one we have in the demo now that includes the fixed part, number, and special character.)  Hopping cycles should have similar administrative flexibility in limiting choices at subscription tiers.  We may limit the low tier to one cycle (daily, weekly, etc) but may wish to adjust the hopping available to upper tiers as an administrative configuration function.

To summarize, I think an administration feature to configure how we set up subscription tiers and what we allow in each tier is preferred if it is possible and easy to code this.  We will like need to do some A/B testing with the subscription tiers and the administrative configuration functionality will be useful.

**Comment by jaygoodswp** [comment]
OK, when I get to the admin console, I'll be sure that these are options.  I've already set things up to limit certain things but some of the other things that you mention have no controls (i.e. limiting to one news source, one title, etc.).
